### ISSUE TITLE
borg mount: TypeError in error handler

### ISSUE THREAD

**Issue by nmiculinic** [borgbackup/borg#1894]
```
borg mount backup-borg/ /mnt/backup                   :(
Enter passphrase for key /mnt/data/gdrive/backup-borg: 
Local Exception
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/borg/archiver.py", line 2769, in main
    exit_code = archiver.run(args)
  File "/usr/lib/python3.5/site-packages/borg/archiver.py", line 2706, in run
    return args.func(args)
  File "/usr/lib/python3.5/site-packages/borg/archiver.py", line 95, in wrapper
    kwargs['manifest'], kwargs['key'] = Manifest.load(repository)
  File "/usr/lib/python3.5/site-packages/borg/helpers.py", line 210, in load
    _, data = key.decrypt(None, cdata)
  File "/usr/lib/python3.5/site-packages/borg/key.py", line 217, in decrypt
    raise IntegrityError('Chunk %s: Encryption envelope checksum mismatch' % bin_to_hex(id))
  File "/usr/lib/python3.5/site-packages/borg/helpers.py", line 880, in bin_to_hex
    return hexlify(binary).decode('ascii')
TypeError: a bytes-like object is required, not 'NoneType'

Platform: Linux protagonist 4.8.10-1-ARCH #1 SMP PREEMPT Mon Nov 21 11:55:43 CET 2016 x86_64 
Linux: arch  
Borg: 1.1.0.dev35+ng701e26a  Python: CPython 3.5.2
PID: 10538  CWD: /mnt/data/gdrive
sys.argv: ['/usr/bin/borg', 'mount', 'backup-borg/', '/mnt/backup']
SSH_ORIGINAL_COMMAND: None

```

I've also tested with 1.0.8 version in arch repositories, same issue.

**Comment by ThomasWaldmann** [comment]
Looks like you have some IntegrityError there and the error handler falls over his own feet.

**Comment by ThomasWaldmann** [comment]
Can you try the patch in PR #1895?

**Comment by ThomasWaldmann** [comment]
Fixed by #1895.
